static/src/js/components/Preferences/PreferencesRadioField.jsx

```diff
@@ -38,10 +38,27 @@ class PreferencesRadioField extends Component {
 
     const {
       enum: values,
-      enumNames,
-      description
+      description,
+      title,
+      enumNames
     } = schema
 
+    let parsedFormData = formData
+    // TODO remove this in a subsequent release
+    if (title === 'Base Layer') {
```

Comment:
When we do the database migration on deployment, if a user is logged in already they have an active jwt token which will contain the old value `blueMarble`. This will be fixed when the user logs out and back in or when the cookie expires. In the meantime we don't want to show blueMarble as an option but, we also need it in the schema so that the user does not see an error message
![image](https://github.com/user-attachments/assets/a7885e09-cd2f-4cdc-a680-b8bf53cd208a)

Reply:
We should do this once only in the `static/src/js/actions/preferences.js` at this level it will push down to other components including this one